appveyor.yml

```diff
@@ -36,8 +40,15 @@ install:
   - rmdir /S /Q storage\mroonga\
   - ps: Get-Content ..\mroonga\version | ForEach-Object { $env:MROONGA_VERSION = $_ }
   - move ..\mroonga storage\mroonga
-  - git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
-  - ps: Get-Content ..\groonga\base_version | ForEach-Object { $env:GROONGA_VERSION = $_ }
+  - if "%USE_MASTER%" == "yes" (
+       git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
+    )
+  - if not "%USE_MASTER%" == "yes" (
```

Comment:
We should use "else".

Reply:
I didn't notice that `else` works as expected. I've fixed.